Engine/Module/ModuleEditor.h

```diff
@@ -59,6 +60,7 @@ class ModuleEditor : public Module
 
 public:
 	GameObject *selected_game_object = nullptr;
+	float3 selected_position = float3(0.0f, 0.0f, 0.0f);
```

Comment:
Because this is a debug variable, it should be deleted now that we have scripting